src/components/WebGLUpload.tsx

```diff
@@ -0,0 +1,361 @@
+import { CLOUDFLARE_URL } from "@/utils/consts";
+import axios from "axios";
+import { useEffect, useState } from "react";
+import { Input } from "./ui/input";
+import { Button } from "./ui/button";
+import { Label } from "./ui/label";
+import { X } from "lucide-react";
+
+// there's probably a better place to put these
+export type BuildFileType = "data" | "framework" | "loader" | "code";
+export const buildFileTypes = Object.freeze({
+  data: "build.data",
+  framework: "build.framework.js",
+  loader: "build.loader.js",
+  code: "build.wasm",
+});
+
+async function uploadBuildFiles(gameId: string, files: Map<string, File>) {
```

Comment:
I don't think you're using this method and unnecessary imports so it can be deleted